realm/realm-library/src/androidTest/java/io/realm/DynamicRealmTests.java

```diff
@@ -287,392 +278,7 @@ public void execute(DynamicRealm realm) {
         } finally {
             RealmLog.remove(testLogger);
         }
-        assertEquals(0, realm.allObjects("Owner").size());
-    }
-
-    @Test
-    public void allObjectsSorted() {
```

Comment:
which test? I could only find tests for `findAllSortedAsync()` not `findAllSorted()`.